ember-app/app/helpers/contrast-color.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export function contrastColor(params) {
+  let value = params[0];
+  return Ember.$.Color('#' + value).contrastColor();
+}
+
+export default Ember.Helper.helper(contrastColor);
```

Comment:
Is this helper actually used? I'm only finding what look like references to the jQuery Color extension.